content/en/docs/setup/additional-setup/gateway/index.md

```diff
@@ -45,6 +45,10 @@ Choose the method you are most familiar with.
 As a security best practice, it is recommended to deploy the gateway in a different namespace from the control plane.
 {{< /tip >}}
 
+All methods listed below rely on [Injection](/docs/setup/additional-setup/sidecar-injection/) to populate additional pod settings at runtime.
+In order to support this, the namespace the gateway is deployed in must not have the `istio-injection=disabled` label.
```

Comment:
This doesn't sound quite right since disabled is the default, so it won't work even if there is no `istio-injection=disabled` label on the namespace. Also, the "Kubernetes YAML" method includes the `sidecar.istio.io/inject: "true"` label on the deployment, so it will work regardless of the namespace label, right?

Reply:
https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#controlling-the-injection-policy. `istio-injection=disabled` overrides any label on the deployment. And because of the workload label, unlabelled namespace is fine